src/main/java/org/n52/wacodis/javaps/algorithms/ForestVitalityChangeAlgorithm.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.n52.wacodis.javaps.algorithms;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import org.esa.snap.core.dataio.ProductIO;
+import org.esa.snap.core.datamodel.Product;
+import org.n52.javaps.algorithm.annotation.Algorithm;
+import org.n52.javaps.algorithm.annotation.ComplexOutput;
+import org.n52.javaps.algorithm.annotation.Execute;
+import org.n52.javaps.algorithm.annotation.LiteralInput;
+import org.n52.javaps.io.GenericFileData;
+import org.n52.wacodis.javaps.WacodisProcessingException;
+import org.n52.wacodis.javaps.command.AbstractCommandValue;
+import org.n52.wacodis.javaps.io.data.binding.complex.GeotiffFileDataBinding;
+import org.n52.wacodis.javaps.io.data.binding.complex.ProductMetadataBinding;
+import org.n52.wacodis.javaps.io.http.SentinelFileDownloader;
+import org.n52.wacodis.javaps.io.metadata.ProductMetadata;
+import org.n52.wacodis.javaps.io.metadata.ProductMetadataCreator;
+import org.n52.wacodis.javaps.io.metadata.SentinelProductMetadataCreator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ *
+ * @author <a href="mailto:s.drost@52north.org">Sebastian Drost</a>
+ */
+@Algorithm(
```

Comment:
A more detailed algorithm description would help the user to understand the process,